packages/flutter/lib/src/widgets/animated_list.dart

```diff
@@ -252,11 +252,27 @@ class AnimatedList extends StatefulWidget {
 /// can refer to the [AnimatedList]'s state with a global key:
 ///
 /// ```dart
+/// // (e.g. in a stateful widget)
 /// GlobalKey<AnimatedListState> listKey = GlobalKey<AnimatedListState>();
+///
 /// ...
```

Comment:
Is `...` special cased somehow? Otherwise, shouldn't this be
```suggestion
/// // ...
```

Reply:
Now that I read the doc in analyze_snippet_code.dart, I see that it is special cased. I would argue that that isn't necessary and instead the `...` should just be added as a comment. Has the added benefit that if I copy this snippet into my own code, I don't get a bunch of analyzer warnings on this line. Everywhere else (e.g. `// continuing from previous example...`) these are also used in comments.

Reply:
yeah, seems reasonable, i'll remove this feature.

Reply:
(Or more to the point, make it require `// ...`. The feature itself is still useful, it adds an ignore.)